eole/utils/optimizers.py

```diff
@@ -41,17 +41,22 @@ def build_torch_optimizer(model, config):
     """
     params = [p for p in model.parameters() if p.requires_grad]
     betas = [config.adam_beta1, config.adam_beta2]
+    print(config.use_amp, optimi_available)
+    if config.use_amp or not optimi_available:
+        optim = torch_optim
+    else:
+        optim = torch_optimi
```

Comment:
`torch_optim` / `torch_optimi` difference not super readable, maybe just use directly `torch.optim` and `optimi`?
The fact that `torch.optim` is also used directly does not help either.